### PR TITLE
doc: Update RPi3 NFS/TFTP setup steps

### PR DESCRIPTION
**Pull request by igoropaniuk**
Update RPi3 NFS/TFTP steps (current steps are out-dated and don't work)
Minor update of Upstream section

`Signed-off-by: Igor Opaniuk <igor.opaniuk@linaro.org>`

### CODE REVIEW

File: docs/rpi3.md [hunk 1]
```diff
@@ -206,44 +206,42 @@ $ sudo rm -rf /srv/nfs/rpi/boot/*
 ```
 
 ### 5.4 Update uboot.env
-We need to make a couple of changes to that file to ensure that it will try to
-boot using everything we have prepared. So, start by inserting the UART cable
-and open up `/dev/ttyUSB0`
-```bash
+Actually, there are two ways to update uboot.env. First, you can edit
```
Comment (jbech-linaro) [inline docs/rpi3.md]: Remove the word "Actually".

File: docs/rpi3.md [hunk 2]
```diff
@@ -206,44 +206,42 @@ $ sudo rm -rf /srv/nfs/rpi/boot/*
...
+`build/rpi3/firmware/uboot.env.txt` file, which is used as simple text source for
+generation of uboot.env during build. And you can just edit u-boot env via UART
```
Comment (jbech-linaro) [inline docs/rpi3.md]: Change from:
` uboot.env during build. And you ...`
to 
` uboot.env during build or you ...`

File: docs/rpi3.md [hunk 3]
```diff
@@ -206,44 +206,42 @@ $ sudo rm -rf /srv/nfs/rpi/boot/*
...
+and save new values to uboot.env. By using the second way you can avoid rebuilding
+and copying uboot.env to SD card, but on the other hand, it's easier to break
```
Comment (jbech-linaro) [inline docs/rpi3.md]: I'd remove from "`,but on the other ...`" to the end of the sentence.

File: docs/rpi3.md [hunk 4]
```diff
@@ -206,44 +206,42 @@ $ sudo rm -rf /srv/nfs/rpi/boot/*
...
+something while performing "hot" edit.
+
+#### 5.4.1 Edit uboot.env.txt
+All you need to do is to edit network configuration in `build/rpi3/firmware/uboot.env.txt`.
+You have to change value of `serverip` to the IP address of your NFS/TFTP server,
+`gatewayip` to your router IP address and `nfspath` to the exported path, where root FS
+is stored (`/srv/nfs/rpi`). Then you need to generate new `uboot.env`:
+```
```
Comment (jbech-linaro) [inline docs/rpi3.md]: I've recently started to add "bash" to code sections to get syntax highlight etc.
```
 "```bash"
```

**Comment by igoropaniuk** [comment]
@jbech-linaro 
Thanks for your comments, fixed

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by igoropaniuk** [comment]
Rebased & applied tags

**Pull request merged**